sdcm/tester.py

```diff
@@ -168,6 +171,7 @@ def __init__(self, methodName='test', name=None, params=None,
         cluster.set_duration(self._duration)
 
         cluster.Setup.set_test_id(self.params.get('test_id'))
```

Comment:
I think we should move this above `self.logdir = os.path.join(cluster.Setup.logdir(), self.id())` or alternativly create new directory for each test run. @roydahan WUT?

Reply:
I don't mind to keep it as is.
DO you see any issue with that?

Reply:
After discussion, we decide that in case of "reuse cluster" we will create a new directory for the run. 
@fruch if it is the case in the code, nothing should be changed.

Reply:
I have tested this case, but I think it won't create a new directory on reuse, but will append to the logs, since it would be in the same directory since it's based on the test_id.

If so we'll want to create one more id for each run, and not based it on the test_id.